### ISSUE TITLE
[SF_ACC] Panel with noAnimation does not have screen reader indication for expanded/collapsed keyboard actions

### ISSUE THREAD

**Issue by Vir-Mittal** [SAP/ui5-webcomponents#6030]
### **Bug Description**
When the panel attribute noAnimation is set to true and the user uses the keyboard to expand/collapse the panel, there is no screen reader announcement. When noAnimation is not set, the screen reader works as expected.

### **Expected Behavior**
The screen reader should announce whether the panel has been expanded/collapsed.

### **Steps to Reproduce**
1. Go to 'https://codesandbox.io/s/panel-no-animation-jyhwgm?file=/index.html'
2. Use NVDA
3. Use the enter key to expand/collapse the panels.
4. Screen reader announcement is present for the panel with animations but not for the panel with no animations.

![Screen Shot 2022-11-09 at 11 20 37 AM](https://user-images.githubusercontent.com/36656306/200922706-e38247c3-2f3f-4f64-8024-06cc2542915f.png)

### **Isolated Example**
https://codesandbox.io/s/panel-no-animation-jyhwgm?file=/index.html

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

**Comment by unazko** [comment]
Hello @SAP/ui5-webcomponents-topic-rl,

The issue is easily reproducible via the provided by the author steps.
The announcements for the expanded and collapsed state of the component come from the `aria-expanded` attribute on the panel header.

Not sure why those announcements aren't present when there is no animation as there is no difference between the two ui5-panel's in the DOM. It seems that this is a timing related issue.

Also the same scenarios are working properly with the sap.m.Panel.

Best Regards,
Boyan
